src/test/utils/getMembers.test.js

```diff
@@ -0,0 +1,131 @@
+  isDisabled,
+  getDisabledMembers,
+  getDisabledOfHousehold,
+  isUnder13,
+  getUnder13Members,
+  getUnder13OfHousehold
+} from '../../utils/getMembers';
+
+// getEveryMember(array, fn) = array.filter(fn)
+test('getEveryMember()', () => {
+  expect(getEveryMember([0, 1, 0], val => val === 0)).toHaveLength(2);
+});
+
+// getEveryMemberOfHousehold(obj, fn) = obj.household.filter(fn)
+test('getEveryMemberOfHousehold()', () => {
+  const client = { household: [0, 1, 0] };
+  expect(getEveryMemberOfHousehold(client, val => val === 0)).toHaveLength(2);
+});
+
+describe('head or spouse', () => {
+  const headOrSpouse = ['head', 'spouse'].map(role => ({ m_role: role }));
+  const notHeadOrSpouse = ['Head', 'member', ''].map(role => ({
+    m_role: role
+  }));
+  const allMembers = headOrSpouse.concat(notHeadOrSpouse);
+
+  test('isHeadOrSpouse()', () => {
+    headOrSpouse.forEach(member => expect(isHeadOrSpouse(member)).toBe(true));
+    notHeadOrSpouse.forEach(member =>
+      expect(isHeadOrSpouse(member)).toBe(false)
+    );
+  });
+
+  // !isHeadOrSpouse()
+  test('isNotHeadOrSpouse()', () => {
+    headOrSpouse.forEach(member =>
+      expect(isNotHeadOrSpouse(member)).toBe(false)
+    );
+    notHeadOrSpouse.forEach(member =>
+      expect(isNotHeadOrSpouse(member)).toBe(true)
+    );
+  });
+
+  test('getHeadOrSpouseMembers()', () => {
+    expect(getHeadOrSpouseMembers(allMembers)).toHaveLength(2);
+  });
+
+  test('getHeadOrSpouseOfHousehold()', () => {
+    const client = { household: allMembers };
+    expect(getHeadOrSpouseOfHousehold(client)).toHaveLength(2);
+  });
+});
+
+describe('dependents', () => {
+  const dependents = [
+    { m_age: 18, m_role: 'member' },
+    { m_age: 17, m_role: 'member' },
+    { m_age: 19, m_disabled: true, m_role: 'member' }
+  ];
+  const notDependents = [
+    { m_age: 18, m_role: 'head' },
+    { m_age: 18, m_role: 'spouse' },
+    { m_age: 19, m_disabled: false, m_role: 'member' },
+    { m_age: 26, m_disabled: false, m_role: 'member' }
+  ];
+  const allMembers = dependents.concat(notDependents);
+
+  test('isDependent()', () => {
+    dependents.forEach(member => expect(isDependent(member)).toBe(true));
+    notDependents.forEach(member => expect(isDependent(member)).toBe(false));
+  });
+
+  test('getDependentMembers()', () => {
+    expect(getDependentMembers(allMembers)).toHaveLength(3);
+  });
+
+  test('getDependentsOfHousehold()', () => {
+    const client = { household: allMembers };
+    expect(getDependentsOfHousehold(client)).toHaveLength(3);
+  });
+});
+
+describe('disabled', () => {
+  const disabled = { m_disabled: true };
+  const notDisabled = { m_disabled: false };
+  const allMembers = [disabled, notDisabled, notDisabled];
+
+  test('isDisabled()', () => {
+    expect(isDisabled(disabled)).toBe(true);
+    expect(isDisabled(notDisabled)).toBe(false);
+  });
+
+  test('getDisabledMembers()', () => {
+    expect(getDisabledMembers(allMembers)).toHaveLength(1);
```

Comment:
What happens if there are no disabled members? I don't even remember what's supposed to happen.

Reply:
I added a test to `getEveryMember()` for this. But it's an empty array.